test-host/README.md

```diff
@@ -0,0 +1,3 @@
+To be used a host runtime for unit tests that involve platform API.
```

Comment:
I haven't found the sdk README get updated with new Profile section :)

Reply:
On my Todo